pkgtools/nls/PKGMAKER.FR

```diff
@@ -84,7 +84,7 @@ HELP.4=seconde ex
 HELP.5=g�n�re de nouveaux fichiers LSM et LST pour le paquet en cons�quence.
 HELP.6=Options :
 HELP.7=
-HELP.8=    $H           Afficher l'aide et quitter
+HELP.8=     $H           Afficher l'aide et quitter

```

Comment:
You sure about the extra space? This is item #8, the next ad future ones are #10+.

Reply:
You're right Jerome, I shouldn't have added it. The latest upload should fix this so you can merge happily.